### PR TITLE
Add project HLTV

### PR DESCRIPTION
**Pull request by s1lentq**
HLTV includes 5 projects that:
- [x] Console
- [x] Core
- [x] Proxy
- [x] DemoPlayer
- [x] Director

### CODE REVIEW

**Comment by In-line** [comment]
Welcome ReHLTV, welcome fake players :unamused: !

File: rehlds/HLTV/Console/src/System.cpp [hunk 1]
```diff
@@ -0,0 +1,1188 @@
...
+	float fps = 0.0f;
+	float nextConsoleUpdate = 0.0f;
+
+	BaseSystemModule::RunFrame(time);
+	while ((m_State == MODULE_RUNNING || m_State == MODULE_CONNECTING))
+	{
+		UpdateTime();
+		double timeDiff = m_SystemTime - m_LastTime;
+		if (m_LastTime > 0 && timeDiff <= 0) {
+			Printf("WARNING! System::RunFrame: system time difference <= 0.\n");
```
Comment (In-line) [inline rehlds/HLTV/Console/src/System.cpp]: Maybe we can use something like c++11 steady_clock ? (For example CLOCK_MONOTONIC in linux)

Comment (WPMGPRoSToTeMa) [inline rehlds/HLTV/Console/src/System.cpp]: It requires static linking of stl library. (but i don't mind it)

Comment (WPMGPRoSToTeMa) [inline rehlds/HLTV/Console/src/System.cpp]: @In-line `Sys_FloatTime` already uses `CLOCK_MONOTONIC` in linux.

Comment (In-line) [inline rehlds/HLTV/Console/src/System.cpp]: @WPMGPRoSToTeMa We need to check win32 implementation too (but it's pain to read it).

Comment (WPMGPRoSToTeMa) [inline rehlds/HLTV/Console/src/System.cpp]: @In-line it uses QPC which is monotonic (`steady_clock` also uses QPC), but there are some another code after QPC call, need to read it.

**Comment by SkillartzHD** [comment]
http://pvpru.com/board/showthread.php?t=520628

what's wrong with this ? or have necessarily to be a brand own ?

**Comment by s1lentq** [comment]
@SkillartzHD what does it have to do with pr?
it is absolutely different things.

**Comment by spark512** [comment]
You guys are awesome :)

**Comment by SkillartzHD** [comment]
@s1lentq 
do you mean that it is a good idea to free a source code with a player who has netchanel attached and all messages svc_ ? etc..  it is enough just to delete the *hltv in connect

**Pull request merged**